CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# [6.24.0](https://github.com/algolia/react-instantsearch/compare/v6.23.3...v6.24.0) (2022-04-19)
+
+
+### Features
+
+* **hooks:** implement <RangeInput> ([2613971](https://github.com/algolia/react-instantsearch/commit/26139714162de7f64d881f9317a391c872fb2316))
```

Comment:
this isn't published yet, so it shouldn't be in the changelog right?

Reply:
Tes that's true, we need to strip them. But in this case this release is irrelevant, no?

Reply:
yes, the release doesn't need to happen I think